scripts/kind-e2e/lib_bash_deploy_kind.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# This should only be sourced
+if [ "${0##*/}" = "lib_helm_deploy_subm.sh" ]; then
```

Comment:
Shouldn't this be "lib_bash_deploy_kind.sh"?

Reply:
Good catch, thanks. Fixed in the latest.